flowpipe/graph.py

```diff
@@ -468,6 +468,40 @@ def node_repr(self):
                 canvas_.add_item(item_)
             x += x_diff
 
+        # Include the input groups if any have been set
+        y_off = 2
```

Comment:
Would it make sense to make these things configurable? Maybe as a module level variable?

Also, the printing code is quite elaborate, usually. Maybe we want to consider moving it to a separate sub-repo as it is not directly related to the functionality?

Reply:
Moving the printing code to somewhere outside is definitely a good idea. I'd move that to a separate ticket where we can then also discuss which parts should be configurable.